<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-FUDE26F" ignored="0" name="Add_02" tests="1" failures="1" timestamp="2020-06-17T22:54:49 ICT" time="0.06" errors="0">
  <testcase name="addStaff1" time="0.023" classname="Test.DepartDaoTest">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Test.DepartDaoTest.addStaff1(DepartDaoTest.java:64)
at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:172)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- addStaff1 -->
</testsuite> <!-- Add_02 -->
